packages/fresh_dio/pubspec.yaml

```
@@ -10,7 +10,7 @@ environment:
   sdk: ">=2.12.0 <3.0.0"
```

```suggestion
  sdk: ">=2.15.0 <3.0.0"
```

That is the min Dio requirement